packages/entity-list/src/components/ListView/ListView.js

```diff
@@ -46,15 +46,23 @@ class ListView extends React.Component {
                 <TableContainer key={`table-${child.id}`} columnDefinitions={getColumnDefinition(child)}/>
               </ListWrapper>
             } else if (actions.isAction(child.componentType)) {
-              return <ActionWrapper key={`listActionWrapper-${child.id}`}>
-                {this.props.showSelectionController && <SelectionControllerContainer/>}
-                <ActionContainer
-                  key={`listAction-${child.id}`}
-                  definition={child}
-                  parent={props.parent}
-                  disabled={this.props.dataLoadingInProgress}
-                />
-              </ActionWrapper>
+              const content = [
+                ...this.props.showSelectionController
```

Comment:
there is `const props = this.props` defined on line 36. many usages over `this.props` in this method not necessary.